_data/current.yml

```diff
@@ -1,3 +1,10 @@
+- name: Squish Days Europe 2019
+  location: Munich, Germany
+  dates: "October 17-18, 2019"
+  url: https://express.converia.de/frontend/index.php?folder_id=2490&page_id=
```

Comment:
This is a very odd / slightly suspicious url. Why don't you have a top level or subdomain?

Reply:
Hi Chris,

That's the website we've used to create the event. It's real! Here's our event announcement on our website: https://www.froglogic.com/news/squish-days-europe-conference-2019/

You can link to that instead. 

Thanks for your help.

Kind regards,

Nick